tests/test_suite.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+
+import argparse
+import os
+import os.path
+import sys
+import time
+
+from lib.log import log
```

Comment:
suggestion: use something more ssg specific than just `lib`. This might cause clashes if the user has another package called `lib`.

`ssg_test_suite`?